pynestml/generated/PyNestMLLexer.py

```diff
@@ -1,16 +1,15 @@
-# Generated from PyNestMLLexer.g4 by ANTLR 4.8
+# Generated from PyNestMLLexer.g4 by ANTLR 4.7.1
 # encoding: utf-8
 from __future__ import print_function
 from antlr4 import *
 from io import StringIO
 import sys
 
 
-
 def serializedATN():
     with StringIO() as buf:
         buf.write(u"\3\u608b\ua72a\u8133\ub9ed\u417c\u3be7\u7786\u5964\2")
```

Comment:
I don't think the unicode marker is required anywhere

Reply:
This file is also auto-generated by Antlr.